recipes/cusparselt/meta.yaml

```
@@ -0,0 +1,93 @@
+  # sha256: 031b75946d2ce22f8c2736f7b4216b9377406afcb35012093a66e2801bfe2990  # [aarch64]
+  # sha256: 88d3e2c662e601214ef5caa26ca2db0492eab6ec3bf9dc9a746bb1a7b8c3aab2  # [win64]
+
+build:
+  number: 0
+  skip: True  # [win32 or cuda_compiler_version in ("None", "9.2", "10.0")]
+  script:
+    - mkdir -p $PREFIX/include                                                # [unix]
+    - cp -r include/* $PREFIX/include/                                        # [unix]
+    - mkdir -p $PREFIX/lib64                                                  # [unix]
+    - mv lib64/{{ cuda_compiler_version }}/libcusparseLt.so* $PREFIX/lib64/   # [unix]
+
+    # - mkdir %LIBRARY_INC%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\include\\cusparseLt.h %LIBRARY_INC%\\                               # [win64]
+    # - mkdir %LIBRARY_INC%\\cusparseLt                                                                                     # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\include\\cusparseLt\\types.h %LIBRARY_INC%\\cusparseLt              # [win64]
+    # - mkdir %LIBRARY_BIN%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\lib64\\{{ cuda_compiler_version }}\\cusparseLt.dll %LIBRARY_BIN%\\  # [win64]
+    # - mkdir %LIBRARY_LIB%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\lib64\\{{ cuda_compiler_version }}\\cusparseLt.lib %LIBRARY_LIB%\\  # [win64]
+  ignore_run_exports_from:
+    - {{ compiler('c') }}    # [linux]
+  run_exports:
+    - {{ pin_subpackage('cusparseLt') }}
+  missing_dso_whitelist:
+    - '*/libgomp.*'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }}
+    - sysroot_linux-64 2.17  # [linux64]
+    - libgomp  # [linux]
+  run:
+    # Only GCC_3.0 or older symbols present
+    - libgcc-ng >=3.0     # [linux]
+    # Only GLIBCXX_3.4 or older symbols present
+    - libstdcxx-ng >=3.4  # [linux]
+  run_constrained:
+    # Only GLIBC_2.17 or older symbols present
+    - __glibc >=2.17      # [linux]
+
+test:
+  commands:
+    - test -f $PREFIX/include/cusparseLt.h                    # [linux]
+    # - test -f $PREFIX/include/cusparseLt/types.h              # [linux]
+    - test -f $PREFIX/lib64/libcusparseLt.so                  # [linux]
+
+    # - if not exist %LIBRARY_INC%\\cusparseLt.h exit 1         # [win64]
+    # - if not exist %LIBRARY_INC%\\cusparseLt\\types.h exit 1  # [win64]
+    # - if not exist %LIBRARY_BIN%\\cusparseLt.dll exit 1       # [win64]
+    # - if not exist %LIBRARY_LIB%\\cusparseLt.lib exit 1       # [win64]
+
+about:
+  home: https://developer.nvidia.com/cusparselt
+  license: cuSPARSELt Software License Agreement
```

Learned this from @beckermr to eliminate the license linter error 🙂
```suggestion
  license: LicenseRef-cuSPARSELt-Software-License-Agreement
```
We probably should do this in the next run of updates for all NVIDIA packages.

Ref: https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=spdx#spdx-identifiers-and-expressions

---

How does this affect how the license is displayed on the Anaconda web server UI?

---

Good question, check this out:
https://anaconda.org/conda-forge/hdf5
They use `LicenseRef-HDF5`.

---

It wouldn't affect the `license_url`'s usage if that's what you are referring to. Think that may come down to some issue with how packages get copied as discussed in issue ( https://github.com/conda-forge/cudatoolkit-feedstock/issues/2 )

---

Looks good. Just want to prevent NVIDIA legal from causing problems :)